packages/larva-tokens/src/brands/variety-vip.json

```diff
@@ -0,0 +1,60 @@
+{
+	"imports": [
+		"../base/aliases.json",
+		"../base/color.json",
+		"../base/font-family.json",
+		"../base/background-color.json"
+	],
+	"props": {
+		"BACKGROUND_COLOR_BRAND_PRIMARY": {
+			"value": "{!BLUE_21}",
+			"type": "color",
+			"category": "background-color"
+		},
+		"BORDER_COLOR_BRAND_PRIMARY": {
+			"value": "{!BLUE_21}",
+			"type": "color",
+			"category": "hr-color"
+		},
+		"COLOR_BRAND_PRIMARY": {
+			"value": "{!BLUE_21}",
+			"type": "color",
+			"category": "text-color"
+		},
+		"COLOR_BRAND_SECONDARY": {
+			"value": "{!RED_46}",
+			"type": "color",
+			"category": "text-color"
+		},
+		"COLOR_BRAND_TERTIARY": {
+			"value": "{!GREY_59}",
+			"type": "color",
+			"category": "text-color"
+		},
+		"FONT_FAMILY_PRIMARY": {
+			"value": "{!SOURCE_SANS_PRO}",
+			"type": "font-family",
+			"category": "font-family"
+		},
+		"FONT_FAMILY_SECONDARY": {
+			"value": "{!BONNIE}",
+			"type": "font-family",
+			"category": "font-family"
+		},
+		"FONT_FAMILY_SECONDARY_FANCY": {
+			"value": "{!IBM_PLEX_SANS}",
+			"type": "font-family",
+			"category": "font-family"
+		},
+		"FONT_FAMILY_BODY": {
+			"value": "{!TIMES_NEW_ROMAN}",
+			"type": "font-family",
+			"category": "font-family"
+		},
+		"FONT_FAMILY_MONO": {
```

Comment:
What is the reason for having IBM_PLEX_MONO as FONT_FAMILY_MONO here vs. FONT_FAMILY_BASIC below? Can they be the same?

Reply:
As I mentioned above, this should be `FONT_FAMILY_ACCENT` for vip and `FONT_FAMILY_BASIC` for variety based on the styleguides, I will update.